DittoHealth/src/androidTest/java/live/ditto/dittoexportlogs/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,24 @@
+package live.ditto.dittoexportlogs
+
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.ext.junit.runners.AndroidJUnit4
+
+import org.junit.Test
+import org.junit.runner.RunWith
+
+import org.junit.Assert.*
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+@RunWith(AndroidJUnit4::class)
+class ExampleInstrumentedTest {
+    @Test
+    fun useAppContext() {
+        // Context of the app under test.
+        val appContext = InstrumentationRegistry.getInstrumentation().targetContext
+        assertEquals("live.ditto.dittoexportlogs.test", appContext.packageName)
```

Comment:
Maybe wrong context/path? perhaps a copy/paste leftover

Reply:
These tests have no values, I will delete the files.
thanks